pytorch_lightning/trainer/configuration_validator.py

```diff
@@ -101,9 +101,7 @@ def check_progress_bar(self, model: "pl.LightningModule") -> None:
         if is_overridden("get_progress_bar_dict", model):
             rank_zero_deprecation(
                 "The `LightningModule.get_progress_bar_dict` method was deprecated in v1.5 and will be removed in v1.7."
-                "Please use the `trainer.progress_bar_callback.get_progress_bar_dict` instead.",
-                stacklevel=5,
-            )
+                "Please use the `trainer.progress_bar_callback.get_progress_bar_dict` instead.")
```

Comment:
This shouldn't mention the Trainer. It should mention only the ProgressBar class